README.md

```diff
@@ -19,6 +19,10 @@
 
 ![DataHub](docs/imgs/datahub-logo.png)
 
+> ✨ DataHub participates in Hacktoberfest. Refer https://hacktoberfest.digitalocean.com for more details.
```

Comment:
Use this emoticon instead? https://emojipedia.org/t-shirt/

Reply:
And https://emojipedia.org/evergreen-tree/ 

( pick a limited edition T-shirt or plant a tree )

Reply:
SGTM